app/controllers/foreman_inventory_upload/accounts_controller.rb

```diff
@@ -21,25 +21,14 @@ def index
       ]
 
       render json: {
-        autoUploadEnabled: Setting[:allow_auto_inventory_upload],
-        hostObfuscationEnabled: Setting[:obfuscate_inventory_hostnames],
-        ipsObfuscationEnabled: Setting[:obfuscate_inventory_ips],
         cloudToken: Setting[:rh_cloud_token],
```

Comment:
Cloud token is also a setting, I wonder if we can also pass it to the new endpoint, or is it just for switchers?

Reply:
Moved it